src/blueprint/from_config.rs

```diff
@@ -168,13 +170,19 @@ fn to_interface_type_definition(definition: ObjectTypeDefinition) -> Valid<Defin
 }
 fn to_fields(type_of: &config::Type, config: &Config) -> Valid<Vec<blueprint::FieldDefinition>> {
   let fields: Vec<Option<blueprint::FieldDefinition>> = type_of.fields.iter().validate_all(|(name, field)| {
+    let field_type = &field.type_of;
+
+    if !is_scalar(field_type) && config.find_type(field_type).is_none() && config.find_union(field_type).is_none() {
+      return Valid::fail(format!("Undeclared type '{field_type}' was found")).trace(&name);
```

Comment:
Performing a `return` will short circuit the execution ignoring all potential errors that could arise in the later part of the checks. For example — if a field has two errors viz no base url and an undeclared type, because we return directly from here, we won't get to know about the base url error until we fix the undeclared type error and then retry.

I would recommend using `validate_or` operator between this logic and the else part so that errors can be combined and shown to the user.

Reply:
Thanks for catching the bug!

Reply:
/tip $100